docs/roman.rst

```diff
@@ -73,7 +73,7 @@ The WFI field of view is laid out as shown in the figure. To select a different
    ['SCA01', 'SCA02', 'SCA03', 'SCA04', 'SCA05', 'SCA06', 'SCA07', 'SCA08', 'SCA09', 'SCA10', 'SCA11', 'SCA12', 'SCA13', 'SCA14', 'SCA15', 'SCA16', 'SCA17', 'SCA18']
    >>> wfi.detector = 'SCA03'
 
-The usable region of the 4096 by 4096 pixel detectors specified for the Wide Field Instrument will range from (4, 4) to (4092, 4092), accounting for the 4 pixel wide bands of reference pixels. **[The preceding may no longer be true.]** To change the position at which to calculate a PSF, simply assign an (X, Y) tuple::
+The usable regions of the Wide Field Instrument's detectors are slightly smaller than their 4096 by 4096 pixel dimensions due to variations in reference pixel coverage at their edges. To change the position at which to calculate a PSF, assign an (X, Y) tuple::
```

Comment:
This text does not seem correct; the reference pixels don't vary. It's a fixed width of 4 pixels around all edges of all detectors. Rephrase to something like the following. 

```
The usable, photosensitive regions of the Wide Field Instrument's detectors are slightly smaller than their 4096 by 4096 pixel dimensions, because the outermost 4 rows and columns are reference pixels that are not sensitive to light. 
```

But that said, I don't understand entirely why this is relevant here for the PSF calculations.  This is a detector property, not something in the optical models.

Reply:
I talked about it with Charles but will catch up with you about this.